package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "fresh",
-  "version": "4.0.1",
-  "description": "Fresh - Bulma + Alpine JS landing page",
+  "name": "Vedran-landing",
+  "version": "0.1.0",
+  "description": "Vedran landing page",
   "main": "gulpfile.js",
   "dependencies": {
     "@ryangjchandler/spruce": "^2.4.0",
     "alpinejs": "^2.8.0",
     "bulma": "^0.9.1",
-    "js-datepicker": "^5.16.0",
-    "notyf": "^3.9.0",
     "codemirror": "^5.58.3",
     "feather-icons": "^4.24.0",
     "ionicons": "^5.2.3",
+    "js-datepicker": "^5.16.0",
+    "notyf": "^3.9.0",
```

Comment:
can you remove dpendencies we don't need? We can do that in separate PR if you would like?